nailgun/entities.py

```diff
@@ -619,6 +638,15 @@ def read(self, entity=None, attrs=None, ignore=()):
             attrs['template_kind'] = {'id': template_kind_id}
         return super(ConfigTemplate, self).read(entity, attrs, ignore)
 
+    def update(self, fields=None):
+        """Fetch a complete set of attributes for this entity.
+
+        FIXME: File a bug at https://bugzilla.redhat.com/ and link to it.
```

Comment:
There are quite a few work-arounds like this. I forgot to file bugs for each of these FIXMEs, though. I'll file a bug asking for these FIXMEs to be taken care of.